posthog/settings.py

```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
     # Add hard-coded feature flags for static self-hosted releases here
     "5440-multivariate-support",
     "new-paths-ui-edge-weights",
+    "sigma-analysis",
+    "new-sessions-player",
```

Comment:
Ok to release @rcmarron ?

Reply:
Yeah, this feature flag has been removed and is live for everyone.